docs/resource/os.md

```diff
@@ -15,31 +15,46 @@ In case of virtualized environments, this is the operating system as it is obser
+| [`os.description`](/docs/attributes-registry/os.md) | string | Human readable (not intended to be parsed) OS version information, like e.g. reported by `ver` or `lsb_release -a` commands. | `Microsoft Windows [Version 10.0.18363.778]`; `Ubuntu 18.04.1 LTS` | `Recommended`                                                                                         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`os.name`](/docs/attributes-registry/os.md)        | string | Human readable operating system name.                                                                                        | `iOS`; `Android`; `Ubuntu`                                         | `Recommended`                                                                                         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`os.version`](/docs/attributes-registry/os.md)     | string | The version string of the operating system as defined in [Version Attributes](/docs/resource/README.md#version-attributes).  | `14.2.1`; `18.04.1`                                                | `Recommended`                                                                                         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
 `os.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
 
-| Value  | Description | Stability |
-|---|---|---|
-| `aix` | AIX (Advanced Interactive eXecutive) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `darwin` | Apple Darwin | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `dragonflybsd` | DragonFly BSD | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `freebsd` | FreeBSD | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `hpux` | HP-UX (Hewlett Packard Unix) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `linux` | Linux | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `netbsd` | NetBSD | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `openbsd` | OpenBSD | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `solaris` | SunOS, Oracle Solaris | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `windows` | Microsoft Windows | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `z_os` | IBM z/OS | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Value          | Description                          | Stability                                                        |
```

Comment:
Please do not make manual changes to auto-generated content of markdown files. These are generated by Weaver based on contents of the yaml model files. Please read the contributing guide: https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#1-modify-the-yaml-model